### PR TITLE
Merge missing bits to get T500RS integration into oversteer

### PR DESCRIPTION
**Pull request by cazzoo**

### CODE REVIEW

File: data/udev/99-thrustmaster-wheel-perms.rules [hunk 1]
```diff
@@ -10,5 +10,8 @@ SUBSYSTEMS=="hid", KERNELS=="0003:044F:B65E.????", DRIVERS=="t500rs", RUN+="/bin
 # Thrustmaster T300RS Racing Wheel (USB)
 SUBSYSTEMS=="hid", KERNELS=="0003:044F:B66E.????", DRIVERS=="t300rs", RUN+="/bin/sh -c 'cd %S%p/../../../; chmod 666 range spring_level damper_level friction_level'"
 
+# Thrustmaster T500RS Racing Wheel (USB)
+SUBSYSTEMS=="hid", KERNELS=="0003:044F:B65E.????", DRIVERS=="t500rs", RUN+="/bin/sh -c 'cd %S%p/../../../; chmod 666 range spring_level damper_level friction_level'"
+
```
Comment (tgurr) [inline data/udev/99-thrustmaster-wheel-perms.rules:15]: This part is already there above at line 7-8.

**Review by tgurr — CHANGES_REQUESTED** [review]

File: data/udev/99-thrustmaster-wheel-perms.rules (hunk 1 above)
Comment (cazzoo) [reply data/udev/99-thrustmaster-wheel-perms.rules:15]: Good catch, i just rebased and didn't check duplicity. Will update now

Comment (tgurr) [reply data/udev/99-thrustmaster-wheel-perms.rules]: Thanks, looks good to me now.

**Comment by berarma** [comment]
Thanks!

**Pull request merged**